### ISSUE TITLE
Error w/ ? operator

### ISSUE THREAD

**Issue by mmstick** [InquisitivePenguin/cascade#3]
So I have this use case that doesn't work, due to the `?` operator:

```rust
let mut data_f =  tempfile()?;
data_f.write_all(&data)?;
data_f.flush()?;
data_f.set_permissions(Permissions::from_mode(0o0644))?;
data_f.seek(io::SeekFrom::Start(0))?;
```

I'd like to write this with cascade, but it will fail until I remove all the `?` operators.

```rust
 let mut data_f =  cascade! {
    tempfile()?;
    ..write_all(&data)?;
    ..flush()?;
    ..set_permissions(Permissions::from_mode(0o0644))?;
    ..seek(io::SeekFrom::Start(0))?;
}

**Comment by snowsignal** [comment]
Hmm, I'll look into why this is happening, though I think I might have a good guess: it didn't expect the `?` token, because as of now, `..` commands only parse functions.

**Comment by snowsignal** [comment]
This has been fixed in version 0.1.3. Sorry for the insane wait, I forgot about this issue.
